src/API/useProgressStatus.js

```diff
@@ -33,19 +36,28 @@ export const useProgressStatus = (id, typeOfProgress, options = {}) => {
     enabled: !!id,
     refetchInterval,
     onSuccess: () => {
-      switch (data?.status) {
```

Comment:
`data.status` can be moved to separate variable

Reply:
IMO switch is more readable in this case, even with type check support

switch (true) {

Reply:
instead of `if` inside `switch` you should have

```
switch true
  case data?.status === JOB_STATUSES.SUCCESSFUL && data?.type === BULK_EDIT_IDENTIFIERS && isProgress: 
     break;
  case data?.status === JOB_STATUSES.SUCCESSFUL:
    break;
  case JOB_STATUSES.FAILED:
    break;
  default:
```

Reply:
@NikitaSedyx  oh in that way. I had some misunderstanding about your prev comment. But for now, it's clear. I added the changes

Reply:
Ye, fixed that, my mistake